pkg/scaffold/controller_test.go

```diff
@@ -15,18 +15,18 @@
 package scaffold
 
 import (
-	"bytes"
 	"testing"
 
 	"github.com/sergi/go-diff/diffmatchpatch"
 )
 
 func TestController(t *testing.T) {
-	codegen := NewControllerCodegen()
-	buf := &bytes.Buffer{}
-	if err := codegen.Render(buf); err != nil {
-		t.Fatal(err)
+	s, buf := setupScaffoldAndWriter()
+	err := s.Execute(appConfig, &Controller{})
+	if err != nil {
+		t.Fatalf("expected nil error, got: (%v)", err)
```

Comment:
maybe change to `t.Fatalf("failed to execute the scaffold: %v", err)` and for all other tests as well.